src/chain/sync.ts

```diff
@@ -103,46 +107,61 @@ export default class ChainSync {
 		this.setupEventListeners()
 
 		// sync the blockchain - effectively backfilling the game state
-		await this.syncBlockchain()
+		await this.syncBlockchain(startFromBlock)
 
 		// change the state to running
 		this._state = State.RUNNING
 	}
 
-	private async syncBlockchain() {
-		Logging.showLogError(`Syncing blockchain...`)
-		// 1. Process all `StakeUpdated` events as this determines who is in the game.
-		const stakeUpdatedFilter = this.stakeRegistry.filters.StakeUpdated()
-		const stakeUpdatedLogs = await this.provider.getLogs({
-			...stakeUpdatedFilter,
-			fromBlock: 7716036, // TODO: set to genesis block for stake registry
-		})
-
-		// now process the logs and add players to the game
-		await this.processStakeUpdatedLogs(stakeUpdatedLogs)
+	private async syncBlockchain(startFromBlock: number) {
+		startFromBlock =
+			Math.floor(startFromBlock / config.blocksPerRound) * config.blocksPerRound
+		Logging.showLogError(`Syncing blockchain from block ${startFromBlock}...`)
+
+		if (false) {
+			// 1. Process all `StakeUpdated` events as this determines who is in the game.
+			const stakeUpdatedFilter = this.stakeRegistry.filters.StakeUpdated()
+			const stakeUpdatedLogs = await this.provider.getLogs({
+				...stakeUpdatedFilter,
+				fromBlock: 7716036, // TODO: set to genesis block for stake registry
+			})
+
+			// now process the logs and add players to the game
+			await this.processStakeUpdatedLogs(stakeUpdatedLogs)
+		}
```

Comment:
Agreed on the visibility.   I've seen the suggestion of "delete it and get it back from git if necessary" in the bee repository, but sometimes I prefer to keep it in-your-face visible for a release or two to be sure it's not likely to resurrect and to prompt me to keep mulling on it.

But yes, this one should be commented just for the visibility because nothing flags the "if (false)" to the human eye.

Reply:
Done, but not yet committed or pushed.